src/Avalonia.Controls.DataGrid/Themes/Default.xaml

```diff
@@ -112,19 +112,22 @@
   </Style>
 
   <Style Selector="DataGridRow /template/ Rectangle#BackgroundRectangle">
-    <Setter Property="IsVisible" Value="False"/>
-    <Setter Property="Fill" Value="#FFBADDE9" />
-  </Style>
-
-  <Style Selector="DataGridRow:pointerover /template/ Rectangle#BackgroundRectangle">
-    <Setter Property="IsVisible" Value="True"/>
-    <Setter Property="Opacity" Value="0.5"/>
-  </Style>
-
-  <Style Selector="DataGridRow:selected /template/ Rectangle#BackgroundRectangle">
-    <Setter Property="IsVisible" Value="True"/>
-    <Setter Property="Opacity" Value="1"/>
-  </Style>
+        <Setter Property="IsVisible" Value="False" />
```

Comment:
from .editorconfig indent size should be 2 spaces

Reply:
Thanks for the info. I edited it in the browser. Made some changes now, is it better?

Reply:
The indent now seems to be 3 spaces instead of 2.

Reply:
No it's two. I click at the top at "File changed" and it's the same indent as others:
![image](https://user-images.githubusercontent.com/14368203/73971640-290cc680-4917-11ea-9180-89662d9394e4.png)